app/src/main/res/layout/view_item_downloads_notify_me.xml

```diff
@@ -19,6 +19,7 @@
     android:id="@+id/downloadsNotifyMe"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
+    android:layout_margin="@dimen/keyline_4"
```

Comment:
Removing the intrinsic margin from the notify me component as it makes it more difficult to have proper margins when designing the screen.

This keyline has been moved to the screens that embed the component